### PR TITLE
Remove ExportToCellH5 module (resolves #2598)

### LINKED ISSUE #2598: Remove ExportToCellH5 module
The ExportToCellH5 module has limited use but requires a number of complex dependencies. We should remove the module from the standard CellProfiler distribution (the ExportToCellH5 module should still be usable by users if they retrieve independent of CellProfiler and copy it to their plugins directory).

### PR DESCRIPTION
**Pull request by 0x00b1**

### CODE REVIEW

**Comment by mcquin** [comment]
https://github.com/CellProfiler/CellProfiler/pull/2599/commits/b83b478645fcabb5b67aacd8747aadbafb8ca580 unnecessary commit?

File: tests/modules/test_exporttocellh5.py [hunk 1]
```diff
@@ -1,322 +0,0 @@
-# '''test_calculatemath.py - Test the CalculateMath module
```
Comment (mcquin) [inline tests/modules/test_exporttocellh5.py]: LMAO

**Comment by 0x00b1** [comment]
@mcquin oops!

**Comment by 0x00b1** [comment]
Fixed!

**Comment by mcquin** [comment]
Needs rebase.

Also: https://github.com/CellProfiler/CellProfiler/issues/2598#issuecomment-279513493

**Pull request merged**